src/process/WindowsLokinetProcessManager.cpp

```diff
@@ -11,16 +11,26 @@
 #include <tlhelp32.h>
 
 // TODO: don't hard-code these paths
-#define LOKINET_DIR "C:\\Program Files\\Loki Project\\Lokinet"
+#define LOKINET_DIR "C:\\Program Files\\Loki Project\\loki-network"
 #define LOKINET_PATH LOKINET_DIR "\\lokinet.exe"
 #define LOKINET_EXE_STR "\"" LOKINET_PATH "\""
+
+WindowsLokinetProcessManager::WindowsLokinetProcessManager()
+{
+    ::CreateMutexA(nullptr, FALSE, "lokinet_qt5_ui");
+}
 
 bool WindowsLokinetProcessManager::doStartLokinetProcess()
-{
+{
+    // try searching one level up from CWD
     bool success = QProcess::startDetached(LOKINET_EXE_STR);
-    if (! success)
-        qDebug("QProcess::startDetached() failed");
-
+    if (! success)
+    {
+        success = QProcess::startDetached("..\\lokinet.exe");

```

Comment:
What's the difference between these two? Does one correspond to the default location where the installer would put it? I'd suggest we support that location only for this release. Later we can try to do something more resilient.

Reply:
so the current installer places the UI in a subfolder of the loki-network path, the hardcoded path above corresponds to early versions of lokinet where the internal package name used to be `loki-network` before being changed to `Lokinet` around the 0.5 release

Reply:
since lokinet 0.5.x, it installs to `$PROGRAMFILES/Loki Project/Lokinet`
anyone who tried lokinet _really_ early on would have installed to `$PROGRAMFILES/Loki Project/loki-network`

Reply:
```suggestion
        // when upgraded from old installations (before 0.5.x), lokinet.exe would be one
        // directory higher
        success = QProcess::startDetached("..\\lokinet.exe");

```

Reply:
Ah, that makes sense.

Reply:
well even _new_ installs will have `lokinet.exe` one level up, just the path to `lokinet.exe` had changed